### PR TITLE
LIVE-2941 LIVE-2942 LLD - route buy and sell to live app + feature flag

### PR DESCRIPTION
**Pull request by LFBarreto**
### 📝 Description

_Redirect all entry points of buy and sell towards the live app if feature flag is on_

### ❓ Context

- **Impacted projects**: `ledger-live-desktop` 
- **Linked resource(s)**: [LIVE-2941] [LIVE-2942] 

### ✅ Checklist

- [ ] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-2941]: https://ledgerhq.atlassian.net/browse/LIVE-2941?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-2942]: https://ledgerhq.atlassian.net/browse/LIVE-2942?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/screens/account/AccountHeaderActions.jsx [hunk 1]
```diff
@@ -277,7 +310,9 @@ const AccountHeaderActions = ({ account, parentAccount, openModal }: Props) => {
       : []),
   ];
 
-  const BuyHeader = <BuyActionDefault onClick={onBuy} />;
+  const BuyHeader = <BuyActionDefault onClick={() => onBuySell("buy")} />;
+
+  const SellHeader = <SellActionDefault onClick={() => onBuySell("sell")} />;
 
   const SwapHeader = <SwapActionDefault onClick={onSwap} />;
```
Comment (gre) [inline apps/ledger-live-desktop/src/renderer/screens/account/AccountHeaderActions.jsx:317]: these variables should start with a lowercase. It's a React$Node, not a React$ComponentType.

File: apps/ledger-live-desktop/src/renderer/screens/asset/AssetBalanceSummaryHeader.jsx [hunk 2]
```diff
@@ -98,16 +103,36 @@ export default function AssetBalanceSummaryHeader({
...
-    history.push({
-      pathname: "/exchange",
-      state: {
-        mode: "onRamp",
-        currencyId: currency.id,
-      },
-    });
-  }, [currency, history]);
+  const onBuySell = useCallback(
+    (mode = "buy") => {
```
Comment (gre) [inline apps/ledger-live-desktop/src/renderer/screens/asset/AssetBalanceSummaryHeader.jsx:107]: is there a reason to make this "mode" a parameter as it's only used one 🤔  (that would simplify things a bit and allow to directly pass the callback and not create inlined fn)

**Comment by ggilchrist-ledger** [comment]
The navigations aren't going to the live app. I have
- the feature flag enabled on prod
- The build from this PR
- Experimental/staging apps available

I'm probably missing something, any ideas? 


https://user-images.githubusercontent.com/87659842/179555368-2963ccdd-6086-46c2-8837-2654fa6d0c67.mov

**Comment by ggilchrist-ledger** [comment]
LGTM

**Pull request merged**